### PR TITLE
about-time 4.2.1 ❄️ 

### PR DESCRIPTION
**Pull request by psteyer**
about-time 4.2.1  :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4420](https://anaconda.atlassian.net/browse/PKG-4420) 
- [Upstream repository](https://github.com/rsalmei/about-time)

### Explanation of changes:

- removing noarch
- adding skip line
- adding script flags
- adding host requirements
- removing python pinnings
- adding about information

[PKG-4420]: https://anaconda.atlassian.net/browse/PKG-4420?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cbouss — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,8 +34,13 @@ test:
 about:
   home: https://github.com/rsalmei/about-time
   summary: Easily measure timing and throughput of code blocks, with beautiful human friendly representations.
+  description: |
+    A cool helper for tracking time and throughput of code blocks, with beautiful human friendly renditions.
   license: MIT
+  license_family: MIT
   license_file: LICENSE
+  doc_url: https://github.com/rsalmei/about-time
+  dev_url: https://github.com/rsalmei/about-time/blob/main/README.md
```
Comment (Jrice1317) [inline recipe/meta.yaml]: Let's switch these (:
```suggestion
  dev_url: https://github.com/rsalmei/about-time
  doc_url: https://github.com/rsalmei/about-time/blob/main/README.md
```

**Review by Jrice1317 — APPROVED** [review]
Besides the flip-flopped urls, LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (psteyer) [reply recipe/meta.yaml]: woops! Nice catch! Thank you

**Review by cbouss — COMMENTED** [review]
abs.yaml is missing. Do publish to snowflake.

**Comment by psteyer** [comment]
Thank you! Added `abs` for snowflake channel. Will merge once built.

**Pull request merged**